CHANGELOG.md

```diff
@@ -0,0 +1,60 @@
+## Widget (2024-05-08)
+
+### Bug Fixes
+
+* disable the confirm wallet button if all required wallets are not selected ([521883f](https://github.com/rango-exchange/rango-client/commit/521883f9e381d311bf0cdad275b234646da2a648))
+
+
+## Widget (2024-04-27)
+
+### Bug Fixes
+
+* make sure to correctly pass validation parameters when setting up a new swap ([4f6d37e](https://github.com/rango-exchange/rango-client/commit/4f6d37ea9b59caca4d0064c1b33b05077b0b59a3))
+
+
+## Widget (2024-04-24)
+
+### Bug Fixes
+
+* make sure to correctly pass validation parameters when setting up a new swap ([4f6d37e](https://github.com/rango-exchange/rango-client/commit/4f6d37ea9b59caca4d0064c1b33b05077b0b59a3))
+
+
+## Widget (2024-04-24)
+
+### Features
+
+* add ethereum for ledger ([084aae2](https://github.com/rango-exchange/rango-client/commit/084aae28adaf0310dffe3a3100dd783252393053))
```

Comment:
I think we could use start each item with capital letter. similar to the react:
https://github.com/facebook/react/blob/main/CHANGELOG.md

Reply:
It makes it hard for who release. It's copy/pasted from other changelog rightnow, if we want to make it capital letter, all the items of list should be edited manually. Besides that, all our changelogs for libraries are lowercase.